Aptoide/aptoide-client-v8
# Feature/asv 1525 appc migration in appview

**What does this PR do?**

This PR aims to add the appc app migration flow to the appview

**Database changed?**

No

**Where should the reviewer start?**

- [ ] AppviewViewManager.java

**How should this be manually tested?**

Install Sky Champ on GPlay. Afterwards, look for the appc version on aptoide (should be in bds-store). You should see the update to appc button in this appview and, when pressed, your GPlay app should be deleted and the new appc app installed;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1525](https://aptoide.atlassian.net/browse/ASV-https://aptoide.atlassian.net/browse/ASV-1525)

**Code Review Checklist**
